packages/suite-desktop/resources/linux/deb/postinst

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env bash
-
-## Electron apps do not have auto-update for deb packages.
-## Instead we add our apt repo as a post-install step. The user can then get app updates
-## via their typical apt update workflow.
-
-# Link the binary into global /usr/bin
-ln -sf '/opt/${productFilename}/${executable}' '/usr/bin/${executable}'
-
-# SUID chrome-sandbox for Electron 5+
-chmod 4755 '/opt/${productFilename}/chrome-sandbox' || true
-
-update-mime-database /usr/share/mime || true
-update-desktop-database /usr/share/applications || true
-
```

Comment:
The lines of code above should be retained as they ensure that the software can be correctly recognized and launched in the browser on Linux.

Reply:
In the Linux system, you can type lichtblick:// in the address bar of the browser to verify my statement. If you delete the above four lines of code, the browser will not be able to correctly trigger the application.

Reply:
Yes it makes sense. As I don't have Linux here to test at the moment, I will restore the files and remove only the APT part